packages/client/src/BrubeckNode.ts

```diff
@@ -55,13 +56,24 @@ export default class BrubeckNode implements Context {
         this.destroySignal.assertNotDestroyed(this)
     }
 
+    private async getNormalizedNetworkOptions(): Promise<NetworkNodeOptions> {
```

Comment:
private async getNormalizedNetworkOptions(): Promise<NetworkNodeOptions> {
        if ((this.options.trackers as TrackerRegistrySmartContract).contractAddress) {
            const trackerRegistry = await getTrackerRegistryFromContract((this.options.trackers as TrackerRegistrySmartContract))
            return {
                ...this.options,
                trackers: trackerRegistry.getAllTrackers()
            }
            // eslint-disable-next-line no-else-return
        } else {
            return this.options as NetworkNodeOptions
        }
    }